packages/shared/src/net/retry.ts

```diff
@@ -38,11 +38,7 @@ export const defaultOptionsRequestNotAccepted: ExecuteWithNetworkRetriesOptions
 const defaultOptions: ExecuteWithNetworkRetriesOptions = {
   ...defaultRetryWithBackoffOptions,
   initialDelay: 1000,
-  httpCodesToRetry: [
-    ...defaultOptionsRequestNotAccepted.httpCodesToRetry,
-    "ECONNRESET", //  (Connection reset by peer): A connection was forcibly closed by a peer. This normally results from a loss of the connection on the remote socket due to a timeout or reboot. Commonly encountered via the http and net modules.
-    AxiosError.ERR_BAD_RESPONSE, // Response cannot be parsed properly or is in an unexpected format.
```

Comment:
Don't  retry on 500 (ERR_BAD_RESPONSE catches it) - and other errors that could lead to reprocessing on non-idempotent endpoints

Reply:
the change is required for all platform, not just this PR - see ticket